spec/lib/auth0/api/v2/rules_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+describe Auth0::Api::V2::Rules do
+  before :all do
+    dummy_instance = DummyClass.new
+    dummy_instance.extend(Auth0::Api::V2::Rules)
+    dummy_instance.extend(Auth0::Mixins::Initializer)
+    @instance = dummy_instance
+  end
+
+  context '.rules' do
+    it { expect(@instance).to respond_to(:rules) }
+    it 'is expected to call get /api/v2/rules' do
+      expect(@instance).to receive(:get).with(
+        '/api/v2/rules', enabled: nil, fields: nil, include_fields: nil, stage: nil)
+      expect { @instance.rules }.not_to raise_error
+    end
+  end
+
+  context '.rule' do
+    it { expect(@instance).to respond_to(:rule) }
+    it 'is expected to call get /api/v2/rules/test' do
+      expect(@instance).to receive(:get).with(
+        '/api/v2/rules/test', fields: nil, include_fields: nil)
+      expect { @instance.rule('test') }.not_to raise_error
+    end
+  end
+
+  context '.create_rule' do
+    it { expect(@instance).to respond_to(:create_rule) }
+    it 'is expected to call post /api/v2/rules' do
+      expect(@instance).to receive(:post).with(
+        '/api/v2/rules',
+        name: 'test', script: 'script', order: 'order', enabled: false, stage: 'login_success')
+      expect do
+        @instance.create_rule('test', 'script', 'order', false)
+      end.not_to raise_error
+    end
+  end
+  context '.update_rule' do
+    it { expect(@instance).to respond_to(:update_rule) }
+    it 'is expected to call put /api/v2/rules/test' do
+      expect(@instance).to receive(:patch).with(
+        '/api/v2/rules/test', script: 'script', order: 'order', enabled: true, stage: 'some_stage')
+      expect do
```

Comment:
Rubocop does not allow to have a multiline block with {}, it has to be moved to do-end. In this case, line is greater that 120 chars, so we keep the do-end.